pkg/api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -20,10 +20,11 @@
 package v1alpha1
 
 import (
-	apiv1beta1 "github.com/aws/eks-anywhere/pkg/providers/snow/api/v1beta1"
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/cluster-api/api/v1beta1"
+
+	apiv1beta1 "github.com/aws/eks-anywhere/pkg/providers/snow/api/v1beta1"
```

Comment:
same question with this, kubebuilder?

Reply:
this was due to `gci`, that fixes imports